frontend/src/common/components/NavBar/Notification/VerificationNotification.tsx

```diff
@@ -0,0 +1,39 @@
+import { NotificationTypes } from "../../../types/NotificationTypes"
+import { showFullDate, showTime } from "../../../utils/FormatUtils"
+
+export default function VerificationNotification({
+  notification
+}: {
+  notification: NotificationTypes
+}) {
+  return (
+    <div className="flex flex-col gap-2">
+      {notification.notificationType === "VERIFICATION" ? (
+        <>
+          <div className="flex gap-1">
+            <div>คุณได้รับการ</div>
+            <div className="font-semibold text-mdd-yellow600">รับรอง</div>
+            <div>จากผู้ดูแล</div>
+          </div>
+          <div>กรุณา log out เพื่อใช้งานในฐานะหมอดู</div>
+        </>
+      ) : (
+        <>
+          <div className="flex gap-1">
+            <div>คุณถูก</div>
+            <div className="font-semibold text-mdd-yellow600">ปฏิเสธ</div>
+            <div>จากผู้ดูแล</div>
+          </div>
+          <div>กรุณาติดต่ออีกครั้งเพื่อยืนยันตัวตน</div>
```

Comment:
- wording มันแปลกๆ นิดนึง น่าจะต้องเป็นประมาณ "กรุณา log in ใหม่อีกครั้งเพื่อใช้งานในฐานะหมอดู" 
- คำถามว่า ถ้าติดต่อใหม่(?) อีกครั้ง นี่ติดต่อไงนะ แล้วมันทำได้ด้วยหรอ